src/include/duckdb/main/settings.hpp

```diff
@@ -65,6 +65,16 @@ struct DebugForceNoCrossProduct {
 	static Value GetSetting(ClientContext &context);
 };
 
+struct DebugOrderedAggregateThreshold {
+	static constexpr const char *Name = "debug_ordered_aggregate_threshold";
```

Comment:
Is this really a debug setting?

Reply:
Apparently not, given how much we are playing with it!